common/workunit/workflow.cpp

```diff
@@ -1093,20 +1083,15 @@ void WorkflowMachine::processSuccessors(CCloneWorkflowItem & item, Owned<IWorkfl
             unsigned numPred = cur.atomicDecNumDependencies();
             if((numPred == 1) && cur.isActive())
             {
-                cur.setAbortFlag(item.queryAbortFlag());
                 addToItemQueue(thisWfid);
             }
         }
         //processLogicalSuccessors made the current function call
         else
         {
+            if(!cur.queryContingencyWithin())
+                cur.setContingencyWithin(item.queryContingencyWithin());//item becomes in date
```

Comment:
not sure what you mean by items becomes in date

Reply:
Please see if my response helps or needs further clarification